### PR TITLE
Add security group

### PR DESCRIPTION
**Pull request by drj11**
As per https://trello.com/c/5FpMkUM2/352-create-a-separate-security-groups-for-elb-ec2-and-rds now have separate security groups for EC2 and RDS instance (and that is separate from the ELB security group).

### CODE REVIEW

File: deployment/stack.py [hunk 1]
```diff
@@ -128,6 +128,7 @@ def main():
         "AutoMinorVersionUpgrade": False,
         "AvailabilityZone": config['AVAILABILITY_ZONE'],
         "BackupRetentionPeriod": "0",
+        "CopyTagsToSnapshot": True,
```
Comment (drj11) [inline deployment/stack.py]: Not required for this PR, but seemed like a good idea.

**Comment by drj11** [comment]
[Create a separate security groups for ELB, EC2 and RDS](https://trello.com/c/5FpMkUM2/352-create-a-separate-security-groups-for-elb-ec2-and-rds)

**Comment by hackdna** [comment]
Just two things:
* EC2 security group: inbound HTTPS from ELB SG only (not from 0.0.0.0/0)
* RDS security group: no outbound traffic to 0.0.0.0/0 ([discussion](http://serverfault.com/questions/812907/aws-security-group-for-rds-outbound-rules))

**Comment by drj11** [comment]
The circularity thing is I think why the first version that worked didn't have Egress rules. Fixing it now.

**Comment by drj11** [comment]
- [ ]  ELB group: no outbound traffic to 0.0.0.0/0 but only for HTTP(80) to EC2 group
- [ ]  EC2 group: no inbound HTTPS(443)
- [ ]  RDS group: no outbound traffic (discussion)

**Comment by drj11** [comment]
I think I lost ilya's comments by force push (I overwrote the last commit).

Ilya maybe we should drop the hack just to get an empty RDS egress rule set. I agree, especially as I was finding it very fiddly to get working (you have to specify a VPC too).

**Pull request merged**